### ISSUE TITLE
Can not train the model

### ISSUE THREAD

**Issue by heshuguo** [ThilinaRajapakse/simpletransformers#879]
**Describe the bug**
Epochs 1/2. Running Loss:    0.0000:  76%
Got the zero running loss, it caused the training stopped.

**To Reproduce**
//download the pre-trained model.
//duing training  at 2 epoch, got the zero running loss, the code stopped.
train_args = {
        "reprocess_input_data": True,
        "overwrite_output_dir": True,
        "num_train_epochs": 2
    }

    # Create a ClassificationModel
    model = ClassificationModel('roberta', 'xxxxxxxx/robera', num_labels=label_length, args=train_args)
    # You can set class weights by using the optional weight argument

    # Train the model
    model.train_model(train_df)

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS

**Additional context**

**Comment by ThilinaRajapakse** [comment]
Please try the solutions described in #860
